config/initializers/rack_profiler.rb

```diff
@@ -0,0 +1,6 @@
+if Rails.env == 'development'
```

Comment:
What about something like

```ruby
if Rails.env == 'development' && ENV['PROFILE'] == 'true'
```

And then if you want to do your profiling your can start rails w/  `PROFILE=true rails s`.

Not sure if rails-config settings can be made available in an initializer (I think I have run into complications with that before), but that might be another option (where a development.local.yml could be used).

It's just a thought, if you have success with a particular pattern I would love to add it to some other projects.